content/source/docs/enterprise/private/migrate.html.md

```diff
@@ -9,6 +9,24 @@ sidebar_current: "docs-enterprise2-private-installer-migration"
 This document outlines the procedure for migrating from the AMI-based Private Terraform Enterprise (PTFE)
 to the Installer-based PTFE.
 
+## Backup
```

Comment:
I think we should add  a small sentence saying that they also need to backup their state file.

Reply:
what state file?  Isn't the state in S3?

Reply:
They should already have a strategy for managing their statefile. I can be more explicit here, but since the product in deployed with terraform, they've already been managing the state file in some way.